data/fields/roof/height.json

```diff
@@ -0,0 +1,6 @@
+{
+    "key": "roof:height",
+    "minValue": 0,
+    "type": "number",
+    "label": "Roof height from facade to top"
```

Comment:
TBH I think the "from facade to top" makes it less clear. The facade is the whole side, right? So does this say from ground to to …?. How can "Roof height" be miss-read?